src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -123,19 +116,23 @@ private function getPersistableClasses(ObjectManager $manager): array
         return $persistableClasses;
     }
 
-    protected function configureIdGenerator(ORMClassMetadataInfo $metadata): void
+    private function saveMetadataToRestore(ClassMetadata $metadata): void
     {
-        $metadata->setIdGeneratorType(ORMClassMetadataInfo::GENERATOR_TYPE_NONE);
-        $metadata->setIdGenerator(new ORMAssignedGenerator());
+        if (!isset($this->metadataToRestore[$metadata->getName()])) {
+            $this->metadataToRestore[$metadata->getName()] = $metadata;
+        }
     }
 
-    private function getMetadata(string $class): ClassMetadata
```

Comment:
This metadata cache is useless has doctrine already has one: https://github.com/doctrine/orm/blob/9a8e1fb47776fc5ff49f995f3fef93a5ff8469b0/lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php#L239-L241

Removed.